contributing/samples.md

```diff
@@ -0,0 +1,257 @@
+## Creating a new sample
+
+Samples live in `examples/source`. Pick one of the existing category folders. 
+
+If your sample can live in a single html file, create a new `*.html` in  one of the sample category folders:
+
+```shell
+$ vim examples/source/1.components/amp-awesome.html
+```
+
+If your sample requires a custom backend API, create a new folder which is going to contain both, your sample html file, which must be named `index.html` and your api implementation `api.js`.
+
+```shell
+$ mkdir examples/source/1.components/amp-awesome
+$ vim examples/source/1.components/amp-awesome/index.html
+$ vim examples/source/1.components/amp-awesome/api.js
+```
+
+If your sample, does not fit into one of the existing categories, please [create an issue](https://github.com/ampproject/docs/issues/new) first and ask for feedback.
+
+## Frontmatter
+
+Samples can define a YAML-based frontmatter. Here is a template to get you started:
```

Comment:
I'm not sure I understand "can define". Must they have a defined YAML-based frontmatter?

Reply:
rephrased it